shell/common/common_benchmarks.sh

```diff
@@ -52,8 +60,9 @@ $0 [-C clusterName <uses aloja_cluster.conf if present or not specified>]
 [-t execution type (e.g: default, experimental)]
 [-e extrae (instrument execution)]
 [-E experiment ID (for use with dashboard)]
+[-B benchmark data size <bytes|[0-9]MB|[0-9]GB|[0-9]TB>
```

Comment:
The suggested usage is more strict than our validator, since it looks like it accepts non-integer arguments. I don't know if we need to support numbers with decimal places, in which case the validator can be simplified. Up to you.

Minor tweak: I would add a `+` after `[0-9]`, since we accept numbers with more than one digit.